powerdnsadmin/default_config.py

```diff
@@ -10,6 +10,7 @@
 HSTS_ENABLED = False
 OFFLINE_MODE = False
 FILESYSTEM_SESSIONS_ENABLED = False
+CSRF_COOKIE_HTTPONLY = True
```

Comment:
The only thing I can see is that people will already have a config file missing this setting. Is it possible to enforce it with a default to 'True' when the app init ?